src/parachain/rewards.ts

```diff
@@ -178,6 +178,39 @@ export class DefaultRewardsAPI implements RewardsAPI {
         );
     }
 
+    async getBlockRewards(
+        vaultAccountId: AccountId,
```

Comment:
I thought we had settled to use strings for the account id so that the UI does not have to do these weird polkadot type conversions? Please change this to a string.

Reply:
I saw we use the accountId also in other places, so I guess fine for now.